### PR TITLE
Fix version number of gRPC.podspec

### PR DESCRIPTION
**Pull request by jcanizales**
And solves a few linter issues that prevent it from being submitted to Cocoapods' repo.

### CODE REVIEW

File: gRPC.podspec [hunk 1]
```diff
@@ -25,16 +24,13 @@ Pod::Spec.new do |s|
...
-                   'Vijay Pai'      => 'vpai@google.com',
-                   'Yang Gao'       => 'yangg@google.com' }
 
     cs.source_files = 'src/core/**/*.{h,c}', 'include/grpc/*.h', 'include/grpc/**/*.h'
     cs.private_header_files = 'src/core/**/*.h'
     cs.header_mappings_dir = '.'
-    cs.xcconfig = { 'HEADER_SEARCH_PATHS' => '"$(PODS_ROOT)/Headers/Build/gRPC" "$(PODS_ROOT)/Headers/Build/gRPC/include"' }
+    cs.xcconfig = { 'HEADER_SEARCH_PATHS' => '"$(PODS_ROOT)/Headers/Build/gRPC" '
+                                             '"$(PODS_ROOT)/Headers/Build/gRPC/include"' }
+    cs.compiler_flags = '-GCC_WARN_INHIBIT_ALL_WARNINGS', '-w'
```
Comment (murgatroid99) [inline gRPC.podspec]: This kind of seems like a bad idea. Why are we suppressing warnings now?

Comment (jcanizales) [inline gRPC.podspec]: Cocoapods' linter won't accept a submission with warnings :(

An alternative is to leave it out and only add it for actually submitting the spec. Same as with the commented-out `s.source` line. On the other hand, those dozens of warnings have been there forever and we still haven't added issues to get them fixed.

Comment (murgatroid99) [inline gRPC.podspec]: For now I think what you have is probably better than adding it only for submission. But we should have issues for fixing those warnings.

Comment (jcanizales) [inline gRPC.podspec]: Ack. Added an A.I. on my calendar for later this week.

**Pull request merged**